charts/csi-wekafsplugin/values.yaml

```diff
@@ -260,6 +260,9 @@ pluginConfig:
     clientGroupName: ""
     # -- Specify NFS protocol version to use for mounting Weka filesystems. Default is "4.1", consult Weka documentation for supported versions
     nfsProtocolVersion: "4.1"
+    # -- Specify name of Weka container to use for mounting filesystems. If not set, container name will be auto-detected
+    # NOTE: for multiple clusters setup, set specific container name rather than attempt to identify it automatically
+    containerName: ""
```

Comment:
rename to frontendContainerName maybe, or wekafsContainerName. since this is a "mountProtocol" section and not a mountProtocol.wekafs (which might also be an option)